src/simulation/dynamics/FuelTank/_UnitTest/test_mass_depletion.py

```diff
@@ -189,6 +189,10 @@ def test_massDepletionTest(show_plots, thrusterConstructor):
         # check a vector values
         np.testing.assert_allclose(dataSigma[i], trueSigma[i], rtol=1e-4, err_msg="Thruster attitude not equal")
 
+    # target value computed from MaxThrust / (EARTH_GRAV * steadyIsp)
```

Comment:
I am a little confused by this comment, because didn't you add the thrust factor to the calculation?

Reply:
I did add thrust factor to the effector's calculation, what I'm saying here is that the test is checking if the thruster has indeed ramped all the way to 100%, so the target value it is checked against is the MaxThrust, not a thrust factor percentage.

My idea was to try to make it clear where the hardcoded allclose value was coming from, but if this isn't clear I can change the wording.

Reply:
This makes sense.